src/components/MarkerPopup/MarkerPopup.jsx

```
@@ -1,33 +1,12 @@
-import React, { useState, useEffect, useRef } from 'react';
+import React, { useState, useEffect} from 'react';
```

ESLint

[error] Insert `·`

(prettier/prettier)